pkg/utils/env.go

```diff
@@ -11,3 +14,12 @@ func GetEnv(envVar string, defaultValue string) string {
 	}
 	return val
 }
+
+// AppendClusterID is utilized to append extra DNS information to the FQDN if a clusterID is configured in the MeshConfig
+// <svc-name>.<svc-namespace> becomes <svc-name>.<svc-namespace>.svc.cluster.<clusterID>
+func AppendClusterID(baseFQDN, clusterID string) string {
+	if clusterID == "" {
+		return baseFQDN
+	}
+	return baseFQDN + fmt.Sprintf(".svc.cluster.%s", clusterID)
+}
```

Comment:
This is only used in `books.go`, so move it there. Also seems out of place in `env.go`, if it needs to be reused since it's k8s specific, it should be in `pkg/k8s`.

Reply:
Moved to `pkg/k8s` as suggested.